### ISSUE TITLE
Argument boxes_flow doesn't work for me (pdf2txt tool)

### ISSUE THREAD

**Issue by jiripraha** [pdfminer/pdfminer.six#540]
The boxes_flow argument doesn't work for me in the pdf2txt tool. Float type, also disabled option.

It helped me:

     if x.lower().strip() == "disabled":
         return x.lower().strip()
     try:
         return float(x)

//--------

            paramv = locals().get(param, None)
            if param == "boxes_flow" and paramv == "disabled":
                 setattr(laparams, param, None)
            elif paramv is not None:
                 setattr(laparams, param, paramv)

Have a nice day

**Comment by pietermarsman** [comment]
Hi @jiripraha,

Unfortunately I don't understand what's not working for you. 

Can you share the command and/or code you are using? Which version of pdfminer.six are you using?

**Comment by jiripraha** [comment]
Hi,
I tried to use the 'boxes_flow' parameter when I ran pdf2txt.py and I think there is an error. I don't dare to pull the request yet, sorry. I'm just getting started with python, but I described the fix above (line 20 and line 46 pdf2txt.py). The parameter is badly processed in pdf2txt and is not used for further processing (both float and disabled options too). Thanks for your patience.
Examples of commands:
_"python pdf2txt.py book.pdf --outfile book.html --output_type html **--boxes-flow disabled** --codec UTF8"
"python pdf2txt.py book.pdf --outfile book.html --output_type html **--boxes-flow 0.9** --codec UTF8"_
